WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -180,7 +180,7 @@ private extension ItemListView {
         static let bannerVerticalPadding: CGFloat = 26
         static let bannerTextSpacing: CGFloat = 4
         static let bannerTitleSpacing: CGFloat = 8
-        static let infoIconInset: EdgeInsets = .init(top: 8, leading: 6, bottom: 8, trailing: 6)
+        static let infoIconInset: EdgeInsets = .init(top: 0, leading: 6, bottom: 0, trailing: 6)
```

Comment:
🗒️ Note for reviewer: at certain font sizes, having top/bottom insets in the info icon results in slight height increase for the item list page header that results in misalignment with the cart page header. The vertical padding is removed here. Maybe we can figure out another way to specify the icon styles, so that each has the expected padding to fit in the container while adjusting size with dynamic typoe.